<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WorkspaceManage</class>
 <widget class="QFrame" name="WorkspaceManage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>356</width>
    <height>645</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="searchframe">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <item>
       <widget class="QLineEdit" name="searchEdit">
        <property name="font">
         <font>
          <family>Arial</family>
          <italic>true</italic>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(152, 152, 152);</string>
        </property>
        <property name="text">
         <string>Search..</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTreeView" name="workspace">
     <property name="minimumSize">
      <size>
       <width>250</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="showDropIndicator" stdset="0">
      <bool>true</bool>
     </property>
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
  <action name="actionTaskNew">
   <property name="text">
    <string>New Task</string>
   </property>
  </action>
  <action name="actionTaskDelete">
   <property name="text">
    <string>Delete Task</string>
   </property>
  </action>
  <action name="actionTaskSetActive">
   <property name="text">
    <string>Set Task Active</string>
   </property>
  </action>
  <action name="actionTaskRename">
   <property name="text">
    <string>Rename</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
